packages/synapse-react-client/src/components/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -188,7 +181,7 @@ export async function extractPlotDataArray(
       plotType === 'BAR'
         ? facetToPlot.facetValues.map(facet => facet.count)
         : undefined,
-    // @ts-ignore
+    // @ts-expect-error
```

Comment:
Prefer `@ts-expect-error` because if some change causes this to become valid, `tsc` will fail and we will know to remove this comment.